Source/MDFastBindingEditor/Private/Customizations/MDFastBindingPropertyBindingExtension.cpp

```diff
@@ -129,17 +130,20 @@ namespace MDFastBindingPropertyBinding
 
 	static void OpenBinding(UWidgetBlueprint* WidgetBlueprint, UMDFastBindingInstance* Binding)
 	{
+		FMDFastBindingEditorModule::OpenBindingEditor(WidgetBlueprint);
```

Comment:
This changes the intended behaviour of using the drawer. I think it should first check if there's already a binding editor tab open, if so use it otherwise open the drawer.

Reply:
I might not be interpreting your thoughts correctly here, but the comment within `FTabManager::InvokeTab_Internal` already indicates that it will try to use an existing tab before opening a new one. Does that address this concern?

Reply:
I was mixing up my plugins and forgot the binding editor tab isn't available in Designer mode (though maybe it should be), but there's still an issue here.

Calling OpenBindingEditor will always create the tab if it's not already open (and switch you to Graph Mode), property binding is done in Designer Mode so this originally opened the Binding Editor Drawer so you could see the new binding, optionally edit it and still keep your place in the designer. With your change, you're forced over to Graph Mode which can be disruptive to the users workflow.

My use case for that is I have a view model plugin that will extend fast binding's property binding extension to expose the view model properties in the property binding dropdown. Creating a binding this way doesn't always require editing the binding so being switched to graph mode would be very disruptive and when it does require editing, that can still be done in the drawer.

Reply:
Alright, I guess I can't really fix this appropriately without having that other plugin also set up. But just to add a couple considerations:

1. Without these changes, clicking on 'Show in Binding Editor' when the binding already exists doesn't work:
![image](https://github.com/DoubleDeez/MDFastBinding/assets/1145663/e93be116-1116-4361-b2f8-ad21d50d4251)

2. I'm not sure if switching over to the graph view is inherently disruptive to one's workflow, if anything it can help with discoverability on where the binding was created. For example, this also happens when trying to bind a delegate:
![image](https://github.com/DoubleDeez/MDFastBinding/assets/1145663/67f19663-3606-496e-8f3c-cc5eaffa2030)

Do you want me to update this PR with just the crash fix, and log an issue for the first point?

Reply:
The other plugin shouldn't be a requirement for a fix but yeah log that issue separately and I'll take a look.

Your point about binding to BP events seems orthogonal to what's going on here since it's a difference between reacting to UI events with non-visual logic and bindng visuals to data.

Reply:
I'm unable to reproduce the issue of the binding editor not opening (see video) - do you know where it's failing for you?

https://github.com/DoubleDeez/MDFastBinding/assets/1893819/fdc9304c-1677-48b0-a915-b9d0ee8da3a9